website/content/en/docs/kubebuilder/references/client.md

```diff
@@ -0,0 +1,574 @@
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *ReconcileApp) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	...
+
+	dep := &v1.Deployment{}
+	err := r.Get(context.TODO(), request.NamespacedName, dep)
+
+	...
+
+	ctx := context.TODO()
+	// A merge patch will preserve other fields modified at runtime.
+	patch := client.MergeFrom(dep.DeepCopy())
+	dep.Spec.Selector.MatchLabels["is_running"] = "true"
+	err := r.Patch(ctx, dep, patch)
+
+	...
+}
+```
+
+[patch-options]:https://godoc.org/sigs.k8s.io/controller-runtime/pkg/client#PatchOption
+[dry-run-all]:https://godoc.org/sigs.k8s.io/controller-runtime/pkg/client#DryRunAll
+[force-ownership]:https://godoc.org/sigs.k8s.io/controller-runtime/pkg/client#ForceOwnership
+
+##### Updating Status Subresource
+
+When updating the [status subresource][cr-status-subresource] from the client, the [`StatusWriter`][status-writer] must be used. The status subresource is retrieved with `Status()` and updated with `Update()` or patched with `Patch()`.
+
+`Update()` takes variadic `client.UpdateOption`'s, and `Patch()` takes variadic `client.PatchOption`'s. See [`Client.Update()`](#update) and [`Client.Patch()`](#patch) for more details. Generally these options are not needed.
+
+##### Status
+
+```Go
+// Status() returns a StatusWriter object that can be used to update the
+// object's status subresource
+func (c Client) Status() (client.StatusWriter, error)
+```
+
+Example:
+```Go
+import (
+	"context"
+	cachev1alpha1 "github.com/example-inc/memcached-operator/pkg/apis/cache/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *ReconcileApp) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	...
+
+	ctx := context.TODO()
+	mem := &cachev1alpha1.Memcached{}
+	err := r.Get(ctx, request.NamespacedName, mem)
+
+	...
+
+	// Update
+	mem.Status.Nodes = []string{"pod1", "pod2"}
+	err := r.Status().Update(ctx, mem)
+
+	...
+
+	// Patch
+	patch := client.MergeFrom(mem.DeepCopy())
+	mem.Status.Nodes = []string{"pod1", "pod2", "pod3"}
+	err := r.Status().Patch(ctx, mem, patch)
+
+	...
+}
+```
+
+[status-writer]:https://godoc.org/sigs.k8s.io/controller-runtime/pkg/client#StatusWriter
+
+#### Delete
+
+```Go
+// Delete deletes the given obj from Kubernetes cluster.
+func (c Client) Delete(ctx context.Context, obj runtime.Object, opts ...client.DeleteOption) error
+```
+
+A `client.DeleteOption` is an interface that sets [`client.DeleteOptions`][delete-opts] fields. A `client.DeleteOption` is created by using one of the provided implementations: [`GracePeriodSeconds`][grace-period-seconds], [`Preconditions`][preconditions], [`PropagationPolicy`][propagation-policy].
+
+Example:
+
+```Go
+import (
+	"context"
+	"k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *ReconcileApp) Reconcile(request reconcile.Request) (reconcile.Result, error) {
```

Comment:
Update for new scaffold. `request reconcile.Request`==>`req ctrl.Request`
Update the imports as well `"sigs.k8s.io/controller-runtime/pkg/reconcile"`==>`ctrl "sigs.k8s.io/controller-runtime"`